docs/userGuide/syntax/panels.md

```diff
@@ -145,6 +145,17 @@ plain text ...
 </variable>
 </include>
 
+<div id = "script_and_styles_warning">
+<box type = "warning" header = "#### Global Effects of the Script and Styles from the Imported Externals">
+
+Importing external resources that contains `script` or `styles` can inadvertently take global effects on your MarkBind website. Due to hoisting during processing, imported scripts and stylesheets affects the entire page. This could potentially alter its appearance and behavior beyond the intended scope.
```

Comment:
> Importing external resources that contains `script` or `styles` can inadvertently take global effects on your MarkBind website. Due to hoisting during processing, imported scripts and stylesheets affects the entire page. This could potentially alter its appearance and behavior beyond the intended scope.

Due to hoisting during processing, imported scripts and stylesheets **affect** the entire page. This could potentially alter the pages appearance and behavior in an unintentional manner.

Reply:
Noted.
Sorry for missing this grammatical error :(